.github/workflows/static-checks.yaml

```diff
@@ -51,6 +51,14 @@ jobs:
         PATH=$PATH:"$HOME/.cargo/bin"
         rustup target add x86_64-unknown-linux-musl
         rustup component add rustfmt clippy
+    - name: Setup seccomp
+      run: |
+        libseccomp_install_dir=$(mktemp -d -t libseccomp.XXXXXXXXXX)
+        gperf_install_dir=$(mktemp -d -t gperf.XXXXXXXXXX)
+        cd ${GOPATH}/src/github.com/${{ github.repository }} && ./ci/install_libseccomp.sh "${libseccomp_install_dir}" "${gperf_install_dir}"
```

Comment:
The two tmp directories passed to `install_seccomp.sh` appear to be unused?!?

Reply:
The two arguments are used at the following points.

`libseccomp_instal_dir`
https://github.com/kata-containers/kata-containers/pull/1788/files#diff-a503dc9420524f1744ca0bf10860e00742fd1b8e131ba412cc3a255a67e776b6R23

`gperf_install_dir`
https://github.com/kata-containers/kata-containers/pull/1788/files#diff-a503dc9420524f1744ca0bf10860e00742fd1b8e131ba412cc3a255a67e776b6R31

Reply:
Ah - I'd missed that. Could you make this a bit clearer? We normally do something like this:

```bash
main()
{
  local arg="${1:-}"

  [ -z "$arg" ] && die "need foo arg"

  # ...
}

main "$@"
```

Reply:
Sorry, I fixed it. 
Thanks.